no-downtime-upgrade/src/main/java/org/keycloak/playground/nodowntimeupgrade/base/model/ObjectModel_V1.java

```diff
@@ -45,4 +46,7 @@
      */
     String getClientTemplateId();
     void setClientTemplateId(String clientTemplateId);
+    
+    String getNode2();
+    void setNode2(String node2);
```

Comment:
What is this node2? There should be no changes needed in the `ObjectModel` interfaces